alleycats-core/src/main/scala/alleycats/Empty.scala

```diff
@@ -79,7 +79,9 @@ object Empty extends EmptyInstances0 {
 
 }
 
-private[alleycats] trait EmptyInstances0 extends compat.IterableEmptyInstance with EmptyInstances1
+private[alleycats] trait EmptyInstances0 extends compat.IterableEmptyInstance with EmptyInstances1 {
+  implicit def optionIsEmpty[A]: Empty[Option[A]] = Empty(None)
```

Comment:
Thanks for the PR!

I see we have `monoidIsEmpty` defined below. I wonder if we could do something similar here, for `MonoidK`. We have an instance of `MonoidK[Option]` so I think it would solve this issue more generally, what do you think?

Reply:
Great idea! See https://github.com/typelevel/cats/pull/4485/commits/ecf02a88a1377f6c8fb820cc9c37df3548898eb0